src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -65,7 +65,7 @@ object Genotype {
       if (r.isNullAt(0))
         -1
       else
-        r.getInt(0)
+        Call.unphasedAlleleRepr(r.getInt(0))
```

Comment:
This should return Call type and this should be just r.getInt(0), no?

Actually, if a pull a few small threads, I think you can basically get rid of unboxedGT (!!!)